File: day3/interface/combine/Fruit.java
public class Fruit { 
    private String shape;
    private String color;
    private String name;
    public Fruit(String fruitShape,String color,String name) {
        shape=fruitShape;
        this.color=color;
        this.name=name;
    }
    public void setShape(String shape) {
        this.shape=shape;
    }
    public String getShape() {
        return this.shape;
    }
    public String toString() {
        return shape+" "+color+" "+name;
    }
}

File: day6/fruit.dart
class Fruit {
  String _color;
  String _shape;
  int _price;
  Fruit(this._color, this._shape, this._price);//unnamed constructor
  Fruit.fruit(this._color, this._shape);//named constructor
   Fruit.lil(this._color);
  @override
  String toString() => "Color :$_color,Shape:$_shape,Price: $_price";
}

main(List<String> args) {
  var fruit = Fruit("red", "round", 100);
  var fruits=Fruit.fruit("orange","flat");
  var fruity=Fruit.lil("yellow");
  print(fruit);
  print(fruits);
  print(fruity);
}
